src/Collections/Sequences/Seq.dfy

```diff
@@ -371,6 +374,7 @@ module Seq {
     requires 0 < |a| && 0 < |b|
     ensures Max(a+b) >= Max(a)
     ensures Max(a+b) >= Max(b)
+    /* Dafny selected trigger: {i in a + b} */
     ensures forall i {:trigger i in [Max(a + b)]} :: i in a + b ==> Max(a + b) >= i
```

Comment:
This seems like an odd choice.  Why are we triggering on the sequence containing the maximum value?